installation-and-configuration/configure-winrm-authentication.md

```diff
@@ -0,0 +1,33 @@
+---
+description: >-
+  This article explains how to configure WinRm authentication on
+  your machine to successfully run snapshots.
+---
+
+# Configure Windows Remote Management (WinRM)
+
+In order to successfully collect data when creating a snapshot, **SysKit Trace** will use a **remote PowerShell session** to connect to the Office 365. This is required to collect the data for Security & Compliance, Exchange Online and some Teams reports.
+
+To make sure that collection can work properly, Windows Remote Management (WinRM) on your computer needs to allow **Basic authentication** (it's enabled by default).
+
+**Multi - Factor Authentication**
+
+Even when using MFA, the WinRM Basic authentication needs to be enabled, because the Basic authentication header is still required to transport the session's OAuth token, since the client-side WinRM implementation has no support for OAuth.
+
+### Enabling Basic Authentication
+
+**1.** To verify that Basic authentication is enabled, run this command in a **Command Prompt**:
+
+`winrm get winrm/config/client/auth`
+
+**2.** If you don't see the value `Basic = true`, you need to run this command to enable Basic authentication for WinRM:
+
+`winrm set winrm/config/client/auth @{Basic="true"}`
```

Comment:
i just remembered, this does not work if the winrm service is not started. perhaps we can add a note about that. it's not required to be started for trace to work, but it is required to execute this command.

Reply:
Added note to make sure service is running